components/treelist/columns/frozen.md

```diff
@@ -105,6 +105,15 @@ If the column you want to freeze is not the first in the list, the treelist must
 }
 ````
 
+## Limitations
```

Comment:
I suggest renaming this section to "Prerequisites" and moving it to the beginning of the article. In my eyes, these seem to be specific requirements rather than limitations.